<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Radiation Therapy Planning Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>10</y>
      <width>330</width>
      <height>287</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="institutionsGroupBox">
         <property name="title">
          <string>Load institutions:</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QPushButton" name="loadDukePushButton">
            <property name="text">
             <string>Duke</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="loadPoconoPushButton">
            <property name="text">
             <string>Pocono</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="loadHighPointPushButton">
            <property name="text">
             <string>High Point</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="loadDukeLineEdit"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="loadPoconoLineEdit"/>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="loadHighPointLineEdit"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="selectQueryCaseGroupBox">
         <property name="title">
          <string>Select Query Case from...</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Duke:</string>
                </property>
                <property name="buddy">
                 <cstring>dukeQueryCaseComboBox</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="dukeQueryCaseComboBox"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Pocono:</string>
                </property>
                <property name="buddy">
                 <cstring>poconoQueryCaseComboBox</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="poconoQueryCaseComboBox"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>High Point:</string>
                </property>
                <property name="buddy">
                 <cstring>highPointQueryCaseComboBox</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="highPointQueryCaseComboBox"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="queryCaseLabel">
           <property name="text">
            <string>Query Case:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="queryCaseNameLabel">
           <property name="text">
            <string>&lt;not selected&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="viewCaseSpaceButton">
         <property name="text">
          <string>View Case Space</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>327</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSelect_Duke_Directory"/>
    <addaction name="actionSelect_Pocono_Directory"/>
    <addaction name="actionSelect_High_Point_Directory"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_View_documentation"/>
    <addaction name="actionAbout_CompareCases"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSelect_Duke_Directory">
   <property name="text">
    <string>Select Duke Directory...</string>
   </property>
  </action>
  <action name="actionSelect_Pocono_Directory">
   <property name="text">
    <string>Select Pocono Directory...</string>
   </property>
  </action>
  <action name="actionSelect_High_Point_Directory">
   <property name="text">
    <string>Select High Point Directory...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionAbout_CompareCases">
   <property name="text">
    <string>About CompareCases</string>
   </property>
  </action>
  <action name="action_View_documentation">
   <property name="text">
    <string>&amp;View documentation</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
